app/src/main/java/com/greenfox/opal/gitinder/response/LoginResponse.java

```diff
@@ -0,0 +1,8 @@
+package com.greenfox.opal.gitinder.response;
+
+public class LoginResponse extends StatusResponse {
+    public LoginResponse(String status, String token) {
```

Comment:
@Garlyle there is no need for these constructors (unless you are using it somewhere directly), just an empty one (json conversion will need that)